README.md

```diff
@@ -1,28 +1,66 @@
+---
+output: github_document
```

Comment:
The output github_document is appearing in a table at the top of the readme on github.

Reply:
Hmm, I can't see it here on the repo home page: https://github.com/Como-DTC-Collaboration/comoDTC

Reply:
This is where I saw it:
https://github.com/Como-DTC-Collaboration/comoDTC/tree/i8-contributing-docs

Reply:
Cool -- thanks. Think this is sorted now.